arches/app/etl_modules/base_import_module.py

```diff
@@ -367,27 +370,147 @@ def start(self, request):
+        validator = FileValidator()
+        if len(validator.validate_file_type(source)) > 0:
+            return {
+                "status": 400,
+                "success": False,
+                "title": _("Invalid excel file/zip specified"),
+                "message": _("Upload a valid excel file"),
+            }
+
+        if source.split(".")[-1].lower() == "zip":
+            with zipfile.ZipFile(source, "r") as zip_ref:
+                files = zip_ref.infolist()
+                for file in files:
+                    if file.filename.split(".")[-1] == "xlsx":
+                        self.cumulative_files_size += file.file_size
+                    if not file.filename.startswith("__MACOSX"):
+                        if not file.is_dir():
+                            result["summary"]["files"][file.filename] = {
+                                "size": (self.filesize_format(file.file_size))
+                            }
+                            result["summary"][
+                                "cumulative_files_size"
+                            ] = self.cumulative_files_size
+                        default_storage.save(
+                            os.path.join(self.temp_dir, file.filename),
+                            File(zip_ref.open(file)),
+                        )
+        elif source.split(".")[-1] == "xlsx":
+            with open(source, "rb") as xlsx_file:
+                result = {
+                    "summary": {
+                        "name": file_name,
+                        "size": self.filesize_format(file_stat.st_size),
+                        "files": {},
+                    }
+                }
+                self.cumulative_files_size += file_stat.st_size
+                result["summary"]["files"][file_name] = {
+                    "size": (self.filesize_format(file_stat.st_size))
+                }
+                result["summary"]["cumulative_files_size"] = self.cumulative_files_size
+                default_storage.save(
+                    os.path.join(self.temp_dir, file_name), File(xlsx_file)
+                )
+
+        has_valid_excel_file = False
+        for file in result["summary"]["files"]:
+            if file.split(".")[-1] == "xlsx":
+                try:
+                    uploaded_file_path = os.path.join(self.temp_dir, file)
+                    workbook = load_workbook(
+                        filename=default_storage.open(uploaded_file_path)
+                    )
+                    self.validate_uploaded_file(workbook)
+                    has_valid_excel_file = True
+                except:
+                    pass
+        if not has_valid_excel_file:
+            title = _("Invalid Uploaded File")
+            message = _(
+                "This file has missing information or invalid formatting. Make sure the file is complete and in the expected format."
+            )
+            return {"success": False, "data": {"title": title, "message": message}}
```

Comment:
If a user were to upload multiple excel files, one valid and one invalid, this would pass without an error. It seems like we should at least provide a message letting the user know that one of their files is invalid if not fail the load entirely.